src/main/java/techcourse/myblog/domain/Article.java

```diff
@@ -1,20 +1,24 @@
 package techcourse.myblog.domain;
 
 import java.util.Objects;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import javax.persistence.*;
+
+import techcourse.myblog.dto.request.ArticleDto;
 
 @Entity
 public class Article {
 	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
 	private Long id;
+
 	private String title;
 	private String contents;
```

Comment:
대량의 문자열은 `@Lob`을 써보면 어떨까요? 그 후, 생성되는 데이터 타입을 확인해 보세요.

```suggestion
	@Lob
	private String contents;
```

Reply:
`@Lob`을 통해서 DB 테이블이 생성될 때 해당 필드의 타입이 **varchar** 타입에서 **longtext** 타입으로 변경되면서 생성되는 것을 확인할 수 있었습니다!